diesel/src/connection/mod.rs

```diff
@@ -8,12 +8,19 @@ use types::HasSqlType;
 
 pub use self::transaction_manager::{TransactionManager, AnsiTransactionManager};
 
+/// To perform simple operations to the backed that implements this trait.
+/// `SimpleConnection` will be used for migrations. Every backend shall implement
+/// their own way to run migrations scripts.
 pub trait SimpleConnection {
-    #[doc(hidden)]
+    /// Brings the ability to execute multiple SQL statements within the same string.
```

Comment:
"Brings the ability to" doesn't really add anything here, and removing this part of the sentence make the rest a nice headline. I'd put the sentences in separate paragraphs too, as the first one can serve as summary and second one is an example.